libs/portals/admin/form-system/src/screens/Admin/Admin.loader.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  FormSystemOrganizationAdmin,
+  FormSystemPermissionType,
+} from '@island.is/api/schema'
+import { WrappedLoaderFn } from '@island.is/portals/core'
+import {
+  GET_ORGANIZATION_ADMIN,
+  LoaderResponse,
+} from '@island.is/form-system/graphql'
+import { Option } from '@island.is/island-ui/core'
+
+export interface AdminLoaderQueryResponse {
+  formSystemOrganizationAdmin: FormSystemOrganizationAdmin
+}
+
+export const adminLoader: WrappedLoaderFn = ({ client, userInfo }) => {
+  return async (): Promise<LoaderResponse> => {
+    const { data, error } = await client.query<AdminLoaderQueryResponse>({
+      query: GET_ORGANIZATION_ADMIN,
+      variables: {
+        input: {
+          nationalId: userInfo?.profile.nationalId,
+        },
+      },
+      fetchPolicy: 'no-cache',
+    })
+    if (error) {
+      throw error
+    }
+    if (!data) {
+      throw new Error('No organization admin was found')
+    }
+
+    const organizationId = data.formSystemOrganizationAdmin
+      ?.organizationId as string
+
+    const organizations = data.formSystemOrganizationAdmin?.organizations?.map(
+      (org) => ({
+        label: org?.label,
+        value: org?.value,
+        isSelected: org?.isSelected,
+      }),
+    ) as Option<string>[]
+
+    const selectedCertificationTypes =
```

Comment:
These can all be extracted at once
```suggestion
    const {organizations, selectedCertificationTypes, etc...} = data.formSystemOrganizationAdmin
```